ld: multiple definition of `random_data_flag' on gcc 10/binutils 2.33.1 on Fedora 32

Link started failing due to multiple definition of `random_data_flag'  on gcc 10/binutils 2.33.1 on Fedora 32.

gcc -Wall -Wextra -Wno-sign-compare -DIPV6 -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection  -Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o fping fping-fping.o fping-seqmap.o fping-socket4.o fping-optparse.o fping-socket6.o  
make[2]: Leaving directory '/builddir/build/BUILD/fping-4.2/src'
/usr/bin/ld: fping-socket4.o:/builddir/build/BUILD/fping-4.2/src/fping.h:14: multiple definition of `random_data_flag'; fping-fping.o:/builddir/build/BUILD/fping-4.2/src/fping.h:14: first defined here
/usr/bin/ld: fping-socket6.o:/builddir/build/BUILD/fping-4.2/src/fping.h:14: multiple definition of `random_data_flag'; fping-fping.o:/builddir/build/BUILD/fping-4.2/src/fping.h:14: first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:350: fping] Error 1
make[1]: *** [Makefile:358: all-recursive] Error 1
make: *** [Makefile:298: all] Error 2

--- a/src/fping.h
+++ b/src/fping.h
@@ -11,7 +11,7 @@
 void crash_and_burn( char *message );
 void errno_crash_and_burn( char *message );
 int in_cksum( unsigned short *p, int n );
-int random_data_flag;
+extern int random_data_flag;
 
 /* socket.c */
 int  open_ping_socket_ipv4();

